packages/dart_frog_cli/pubspec.yaml

```
@@ -8,7 +8,7 @@ documentation: https://dartfrog.vgv.dev/docs/overview
 topics: [server, backend, shelf, dart-frog, cli]
 
 screenshots:
-  - description: 'Dart Frog logo.'
+  - description: "Dart Frog logo."
```

We should revert this change:
```suggestion
  - description: 'Dart Frog logo.'
```

---

check [commit](https://github.com/VeryGoodOpenSource/dart_frog/pull/924/commits/6917ad91d6cc7dd375def6d975f216246855b625)